astroid/__init__.py

```diff
@@ -165,6 +167,15 @@
 
 from astroid.util import Uninferable
 
+# Performance hack for tokenize. See https://bugs.python.org/issue43014
+# Adapted from https://github.com/PyCQA/pycodestyle/pull/993
+if (
+    not PY310_PLUS
+    and callable(getattr(tokenize, "_compile", None))
+    and getattr(tokenize._compile, "__wrapped__", None) is None  # type: ignore[attr-defined]
+):  # pragma: no cover
```

Comment:
I think this should get covered by us right? We are testing on `3.10`. Or am I missing something?